hexrdgui/masking/mask_manager.py

```diff
@@ -28,6 +28,16 @@ def __init__(self, name='', mtype='', visible=True):
         self.name = name
         self.visible = visible
         self.masked_arrays = None
+        self.last_view_mode = ViewType.raw
```

Comment:
This might be better named `masked_arrays_view_mode`, since it is being used specifically to keep track of the current view mode of the `masked_arrays`.

Reply:
Got it, changed :+1: